BookingsAPI/Bookings.IntegrationTests/Helper/TestDataManager.cs

```diff
@@ -113,7 +143,7 @@ private CaseType GetCaseTypeFromDb(string caseTypeName)
 
                 if (caseType == null)
                 {
-                    throw new InvalidOperationException("Unknown case type: "  + caseTypeName);
+                    throw new InvalidOperationException("Unknown case type: " + caseTypeName);
```

Comment:
should this be "ArgumentNullException" ??

Reply:
It, s not actually argument null